sympy/solvers/solveset.py

```diff
@@ -192,6 +191,11 @@ def inv(trig):
                 invs += Union(*[imageset(Lambda(n, L(g)), S.Integers) for g in g_ys])
             return _invert_real(f.args[0], invs, symbol)
 
+    if (isinstance(a,log) for a in f.args):
```

Comment:
> seems that this is a tuple

Just a point of information, it is a generator -- a lazy list comprehension -- which is always True as you have said.